pyItunes/Library.py

```diff
@@ -110,6 +110,10 @@ def getSongs(self):
 				s.has_video = attributes.get('Has Video')
 			if attributes.get('Grouping'):
 				s.grouping = attributes.get('Grouping')
+
+			s.loved = True if attributes.get('Loved') else False
+			s.album_loved = True if attributes.get('Album Loved') else False
```

Comment:
Note I'm doing these as one-liners, and providing a default value. Later I'll do a PR that provides default values for all of the attributes (not providing defaults forces people integrating this dependency to do tons of if/else checks).